GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.cpp

```diff
@@ -408,6 +408,34 @@ RenderData* Renderer::post_effect_render_data()
     return post_effect_render_data_;
 }
 
+    RenderData* Renderer::post_effect_render_data_vulkan()
```

Comment:
First, try to have common for vulkan and GL. if not possible, keep them in respective renderers with abstract method in renderer.